reference/commands/graph/info.rst

```diff
@@ -130,5 +130,379 @@ The ``conan graph info`` command shows information about the dependency graph fo
 
 .. warning::
 
-  The json output of the ``conan graph --format=json`` is **experimental** and subject to
+  The json output of the ``conan graph info --format=json`` is **experimental** and subject to
   change.
+
+
+**Examples**:
+
+.. code-block:: bash
+
+    $ conan info .
+    $ conan info myproject_folder
+    $ conan info myproject_folder/conanfile.py
+    $ conan info hello/1.0@user/channel
+
+The output will look like:
+
+.. code-block:: text
+
+    $ conan graph info recipes/binutils/all/conanfile.py --version=2.38
```

Comment:
This is not clear from where it comes (conancenter, I know). Maybe use a ``--requires`` or define a local ``conanfile.py`` as example?